src/lib/one-graph/cli-client.js

```diff
@@ -289,15 +388,45 @@ const writeLockfile = ({ lockfile, siteRoot }) => {
 }
 
 /**
- * Read the Netlify Graph lockfile from disk, if it exists
+ * Persist the Netlify Graph lockfile on disk
  * @param {object} input
  * @param {string} input.siteRoot The GraphQL schema to use when generating code
- * @return {NetlifyGraphLockfile.V0_format | undefined}
+ * @param {string=} input.schemaId
+ * @param {string=} input.operationsHash
  */
-const readLockfile = ({ siteRoot }) => {
+const mergeLockfile = ({ operationsHash, schemaId, siteRoot }) => {
+  const lockfile = readLockfile({ siteRoot })
+  if (lockfile) {
+    /** @type {NetlifyGraphLockfile.V0_format} */
+    const newLockfile = {
```

Comment:
this copy is probably unnecessary

Reply:
Just giving it a name and making things explicit